libs/ruby.json

```diff
@@ -14,5 +14,25 @@
   {"id": "rb.paypal-sdk-rest", "imports": ["paypal-sdk-rest"], "name": "paypal-sdk-rest", "repo": "paypal/PayPal-Ruby-SDK", "tags": ["payments"], "tech": ["e-commerce"]},
   {"id": "rb.mustache", "imports": ["mustache"], "name": "mustache", "repo": "mustache/mustache", "tags": ["templates"], "tech": ["ruby-web"]},
   {"id": "rb.mini-magick", "imports": ["MiniMagick"], "name": "MiniMagick", "repo": "minimagick/minimagick", "tags": ["manipulation"], "tech": ["images"]},
-  {"id": "rb.rmagick", "imports": ["RMagick"], "name": "RMagick", "repo": "rmagick-temp/rmagick", "tags": ["manipulation"], "tech": ["images"]}
+  {"id": "rb.rmagick", "imports": ["RMagick"], "name": "RMagick", "repo": "rmagick-temp/rmagick", "tags": ["manipulation"], "tech": ["images"]},
+
+  {"id": "rb.active-interaction", "imports": ["active_interaction"], "name": "ActiveInteraction", "repo": "AaronLasseigne/active_interaction", "tags": ["framework"], "tech": ["app-arch"]},
+  {"id": "rb.apotomo", "imports": ["apotomo"], "name": "Apotomo", "repo": "apotonick/apotomo", "tags": ["web-components"], "tech": ["app-arch"]},
+  {"id": "rb.cells", "imports": ["cells"], "name": "Cells", "repo": "trailblazer/cells", "tags": ["view-components"], "tech": ["app-arch"]},
+  {"id": "rb.decent-exposure", "imports": ["decent_exposure"], "name": "Decent Exposure", "repo": "hashrocket/decent_exposure", "tags": ["interfaces-for-controllers"], "tech": ["app-arch"]},
+  {"id": "rb.docile", "imports": ["docile"], "name": "Docile", "repo": "ms-ati/docile", "tags": ["dsl-wrapper"], "tech": ["utils"]},
+  {"id": "rb.dry-view", "imports": ["dry_view"], "name": "dry-view", "repo": "dry-rb/dry-view", "tags": ["rendering"], "tech": ["app-arch"]},
+  {"id": "rb.dry-schema", "imports": ["dry_schema"], "name": "dry-schema", "repo": "dry-rb/dry-schema", "tags": ["dsl-schema"], "tech": ["utils"]},
+  {"id": "rb.dry-system", "imports": ["dry_system"], "name": "dry-system", "repo": "dry-rb/dry-system", "tags": ["components"], "tech": ["app-arch"]},
+  {"id": "rb.dry-container", "imports": ["dry-container"], "name": "dry-container", "repo": "dry-rb/dry-container", "tags": ["containers"], "tech": ["app-arch"]},
+  {"id": "rb.dry-matcher", "imports": ["dry-matcher"], "name": "dry-matcher", "repo": "dry-rb/dry-matcher", "tags": ["pattern-matching"], "tech": ["funcional"]},
```

Comment:
Hm... i know it's what we did in past but i am not convinced that pattern matching belongs to functional. let's keep it there for now but hm...